File: sdks/thunkmetrc-cs/ThunkMetrc.Wrapper/Models/PackagesUpdateRemediateRequestItem.cs
using System;
using System.Collections.Generic;

namespace ThunkMetrc.Wrapper.Models
{
    public class PackagesUpdateRemediateRequestItem
    {
        [global::System.Text.Json.Serialization.JsonPropertyName("PackageLabel")]
        public string? PackageLabel { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("RemediationDate")]
        public string? RemediationDate { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("RemediationMethodName")]
        public string? RemediationMethodName { get; set; }
        [global::System.Text.Json.Serialization.JsonPropertyName("RemediationSteps")]
        public string? RemediationSteps { get; set; }
    }
}

File: sdks/thunkmetrc-go/wrapper/models/packages_update_remediate_request_item.go
package models

type PackagesUpdateRemediateRequestItem struct {
    PackageLabel string `json:"PackageLabel,omitempty"`
    RemediationDate string `json:"RemediationDate,omitempty"`
    RemediationMethodName string `json:"RemediationMethodName,omitempty"`
    RemediationSteps string `json:"RemediationSteps,omitempty"`
}
